peekingduck/pipeline/nodes/output/screen.py

```diff
@@ -33,17 +33,35 @@ class Node(AbstractNode):
         |pipeline_end|
 
     Configs:
-        None.
+        window_name (:obj:`str`): **default = "PeekingDuck"** |br|
+            Name of the displayed window.
+        window_size (:obj:`Dict`):
+            **default = { do_resizing: False, width: 1280, height: 720 }** |br|
+            Resizes the displayed window to the chosen width and weight, if ``do_resizing``
+            is set to ``true``.
+        window_loc (:obj:`Dict`): **default = { x: 0, y: 0 }** |br|
+            X and Y coordinates of the top left corner of the displayed window, with
+            reference from the top left corner of the screen, in pixels.
     """
 
     def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
+        cv2.namedWindow(self.window_name, cv2.WINDOW_NORMAL)
+        cv2.moveWindow(self.window_name, self.window_loc["x"], self.window_loc["y"])
 
     def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
         """Show the outputs on your display"""
-        cv2.imshow("PeekingDuck", inputs["img"])
+
+        if not self.window_size["do_resizing"]:
+            cv2.imshow(self.window_name, inputs["img"])
+        else:
+            resized_img = cv2.resize(
+                inputs["img"], (self.window_size["width"], self.window_size["height"])
+            )
+            cv2.imshow(self.window_name, resized_img)
```

Comment:
Would like to propose streamlining the code segment in lines 55-61 as:
```
img = inputs["img"]
if self.window_size["do_resizing"]:
    img = cv2.resize(img, (self.window_size["width"], self.window_size["height"]))
cv2.imshow(self.window_name, img)
```

Reply:
This looks better, updated.